xmtp_mls/IDENTITY.md

```diff
@@ -54,6 +54,7 @@ XMTP installations consist of a long-lived Ed25519 key-pair (the 'installation k
    XMTP: Grant Messaging Access
 
    Current Time: <ISO 8601 date and time with local UTC offset>
+   Account Address: <ethereum address>
    Installation ID: <hex(last_20_bytes(keccak256(Ed25519PublicKey)))>
```

Comment:
@jhaaaa @saulmc - would love your feedback on the verbiage in the signature text. Q's:

- Should we add text explaining what is being granted, what the installation ID is used for, what kind of verification the user needs to do? We need the user to verify the 'current time' is correct, and that they are signing for the correct 'account address'. The installation ID is not a secret and they don't need to memorize it. The 'installation ID' and 'current time' are there so they can list installations in the future and recognize this one, revoking it if needed.
- Should we change the labels for clarity, for example `Granted Time` instead of `Current Time`, `Installation Token` or `Granted Token` etc instead of `Installation Id`?
- Ethereum address are formatted as `0x<random string>`, should installation Id's be formatted differently for clarity, e.g. `x:<Random String>`?

Reply:
Merging this to unblock others, but will keep track of this thread and add changes separately!

Reply:
Defer to @jhaaaa 's expertise on these questions!